metrics/storage/blogbench_dockerfile/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # Set up an Ubuntu image with 'blogbench' installed
 
 # Usage: FROM [image name]
-FROM ubuntu
+FROM quay.io/libpod/ubuntu
```

Comment:
this image is not up-to-date, podman folks recommended me not to use it, can you try with this image public.ecr.aws/lts/ubuntu:20.04_stable ?

Reply:
@devmic updated the change.